test/GPv2TradeExecution.test.ts

```diff
@@ -1,315 +0,0 @@
-        ).to.not.be.reverted;
-      });
-
-      it("should revert when ERC20 transfer returns false", async () => {
-        const amount = ethers.utils.parseEther("4.2");
-
-        const sellToken = await waffle.deployMockContract(deployer, IERC20.abi);
-        await sellToken.mock.transferFrom
-          .withArgs(traders[0].address, recipient.address, amount)
-          .returns(false);
-
-        await expect(
-          tradeExecution.transferSellAmountToRecipientTest(
-            {
-              owner: traders[0].address,
-              sellToken: sellToken.address,
-              sellAmount: amount,
-              ...withoutBuy,
-            },
-            recipient.address,
-          ),
-        ).to.be.revertedWith("failed transferFrom");
-      });
-    });
-  });
-
-  describe("transferBuyAmountToOwner", () => {
-    const withoutSell = {
-      sellToken: ethers.constants.AddressZero,
-      sellAmount: ethers.constants.Zero,
-    };
-
-    it("should transfer buy amount to receiver", async () => {
-      const [owner, receiver] = traders;
-
-      const amount = ethers.utils.parseEther("13.37");
-
-      const buyToken = await waffle.deployMockContract(deployer, IERC20.abi);
-      await buyToken.mock.transfer
-        .withArgs(receiver.address, amount)
-        .returns(true);
-
-      await expect(
-        tradeExecution.transferBuyAmountToOwnerTest({
-          owner: owner.address,
-          receiver: receiver.address,
-          buyToken: buyToken.address,
-          buyAmount: amount,
-          ...withoutSell,
-        }),
-      ).to.not.be.reverted;
-    });
-
-    it("should allow receiver to be same as the owner", async () => {
-      const amount = ethers.utils.parseEther("13.37");
-
-      const buyToken = await waffle.deployMockContract(deployer, IERC20.abi);
-      await buyToken.mock.transfer
-        .withArgs(traders[0].address, amount)
-        .returns(true);
-
-      await expect(
-        tradeExecution.transferBuyAmountToOwnerTest({
-          owner: traders[0].address,
-          receiver: traders[0].address,
-          buyToken: buyToken.address,
-          buyAmount: amount,
-          ...withoutSell,
-        }),
-      ).to.not.be.reverted;
-    });
-
-    it("should revert on failed ERC20 transfers", async () => {
-      const amount = ethers.utils.parseEther("4.2");
-
-      const buyToken = await waffle.deployMockContract(deployer, IERC20.abi);
-      await buyToken.mock.transfer
-        .withArgs(traders[0].address, amount)
-        .revertsWithReason("test error");
-
-      await expect(
-        tradeExecution.transferBuyAmountToOwnerTest({
-          owner: traders[0].address,
-          receiver: traders[0].address,
-          buyToken: buyToken.address,
-          buyAmount: amount,
-          ...withoutSell,
-        }),
-      ).to.be.revertedWith("test error");
-    });
-
-    it("should not revert when transfering from a token with no contract at its address", async () => {
```

Comment:
Same question for this test.

Reply:
https://github.com/gnosis/gp-v2-contracts/blob/37f5a84b3664f76112f5e0e4a8fc31feb6f540c7/test/GPv2SafeERC20.test.ts#L244